pkgs/top-level/ocaml-packages.nix

```diff
@@ -567,14 +626,26 @@ let
 
     hmap = callPackage ../development/ocaml-modules/hmap { };
 
+    hol_light = callPackage ../applications/science/logic/hol_light { }; # App from all-packages
```

Comment:
I’d rather see this out of this set. Currently, the few packages that should be kicked out of here are conveniently gathered at the end. With this change, they get lost.

Reply:
Done